models/gemini/models/llm/llm.py

```diff
@@ -445,6 +449,14 @@ def _handle_generate_stream_response(
                                 ),
                             )
                         ]
+                    elif part.inline_data:
+                        base64_str = base64.b64encode(part.inline_data.data).decode('utf-8')
+                        assistant_prompt_message.content = [MultiModalPromptMessageContent(
```

Comment:
I have updated the implementation to append an ImagePromptMessageContent instead.

The `MultiModalPromptMessageContent` serves as an abstraction for multimodal support and should not be used directly. It is recommended to utilize subclasses of `MultiModalPromptMessageContent` for specific implementations.

Reply:
Got it, I use `ImagePromptMessageContent` for now and wait for the main repository to update.